tools/deb/envoy_bootstrap_tmpl.json

```diff
@@ -0,0 +1,121 @@
+{
+  "stats_config": {
+    "use_all_default_tags": false
+  },
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": {
+      "socket_address": {
+        "address": "127.0.0.1",
+        "port_value": 15000
+      }
+    }
+  },
+  "dynamic_resources": {
+    "lds_config": {
+      "api_config_source": {
+        "api_type": "REST_LEGACY",
+        "refresh_delay": {{ .refresh_delay }},
+        "cluster_names": [
+          "rds"
```

Comment:
why is LDS pointing to RDS cluster?

Reply:
LDS v1 hardcodeds "rds" for clusters - in v2 we have a single cluster for all pilot connections, 
so until we move to LDS v2 we'll use the 'rds' name for the cluster. After that we can name it 'pilot' or whatever else.

Reply:
Looking at Looking at https://github.com/istio/istio/blob/master/pilot/pkg/proxy/envoy/v1/testdata/envoy-sidecar-auth.json.golden#L33 I see different clusters for xDS services.

Reply:
Sure - v1 used multiple clusters, in v2 we use one.